vector/src/main/java/im/vector/app/features/form/FormEditTextItem.kt

```diff
@@ -80,10 +85,11 @@ abstract class FormEditTextItem : VectorEpoxyModel<FormEditTextItem.Holder>() {
 
         holder.textInputEditText.isEnabled = enabled
         inputType?.let { holder.textInputEditText.inputType = it }
-        holder.textInputEditText.isSingleLine = singleLine ?: false
+        holder.textInputEditText.isSingleLine = singleLine
```

Comment:
I've seen that singleline is deprecated in xml, is it still ok to do it programmatically?

Reply:
this is your code :)